services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_events_utils.py

```diff
@@ -230,29 +230,13 @@ async def service_remove_sidecar_proxy_docker_networks_and_volumes(
             task_progress.update(
                 message="removing volumes", percent=ProgressPercent(0.3)
             )
-            unique_volume_names = [
-                DynamicSidecarVolumesPathsResolver.source(
-                    path=volume_path,
-                    node_uuid=scheduler_data.node_uuid,
-                    run_id=scheduler_data.run_id,
-                )
-                for volume_path in [
-                    DY_SIDECAR_SHARED_STORE_PATH,
-                    scheduler_data.paths_mapping.inputs_path,
-                    scheduler_data.paths_mapping.outputs_path,
-                    *scheduler_data.paths_mapping.state_paths,
-                ]
-            ]
-            with log_context(
-                _logger, logging.DEBUG, f"removing volumes via service for {node_uuid}"
-            ):
+            with log_context(_logger, logging.DEBUG, f"removing volumes '{node_uuid}'"):
+                rabbit_rpc_client: RabbitMQRPCClient = app.state.rabbitmq_rpc_client
                 await remove_volumes_from_node(
```

Comment:
Q: so you ask the agent to remove the volumes from one of the nodes. Every node has one simcore agent service (was setup as `global`)  how do you know that the correct agent is taking this task?